ubuntu/16.04/usr/local/share/container/baseimage-10.sh

```diff
@@ -23,7 +23,7 @@ do_build() {
 }
 
 check_development_start() {
-  if [ "$DEVELOPMENT_MODE" -eq 0 ]; then
+  if [ "$DEVELOPMENT_MODE" == 'true' ]; then
```

Comment:
Conversions from `0`/`1` to `true`/`false` are now in place. What does YAML pass through boolean `true`/`false` as?